probe/websocket/ws_test.go

```diff
@@ -50,6 +51,21 @@
 		}
 	}()
 
+	// wait for http server to start
```

Comment:
Add a comment explaining why the waiting mechanism is necessary, as it might not be immediately clear to other developers. [medium]